recipes/aocl-blas/recipe.yaml

```
@@ -0,0 +1,58 @@
+schema_version: 1
+
+context:
+  version: 5.1
+
+package:
+  name: aocl-blas
+  version: ${{ version }}
+
+source:
+    url: https://github.com/amd/blis/archive/refs/tags/${{ version }}.tar.gz
+    sha256: 4ab210cea8753f4be9646a3ad8e6b42c7d19380084a66312497c97278b8c76a4
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - ${{ stdlib("c") }}
+    - ${{ compiler("c") }}
+    - ${{ compiler("cxx") }}
+    - if: not win
+      then: ${{ compiler("fortran") }}
+    - if: not linux
+      then: llvm-openmp
+    - if: linux
+      then: libgomp
+    - if: not win
+      then: make
+    - if: win
+      then: ninja
+    - cmake
+  host:
+  run:
+
+tests:
+  - script:
+      - if: unix
+        then: test -f $PREFIX/lib/libblis-mt${SHLIB_EXT}
+      - if: unix
+        then: test -f $PREFIX/include/blis.h
+      - if: win
+        then: if not exist %LIBRARY_INC%\\blis.h exit 1
+      - if: win
+        then: if not exist %LIBRARY_LIB%\\AOCL-LibBlis-Win-MT-dll.dll exit 1
```

```suggestion
      - if: unix
        then: 
          - test -f $PREFIX/lib/libblis-mt${SHLIB_EXT}
          - test -f $PREFIX/include/blis.h
      - if: win
        then: 
          - if not exist %LIBRARY_INC%\\blis.h exit 1
          - if not exist %LIBRARY_BIN%\\AOCL-LibBlis-Win-MT-dll.dll exit 1
          - if not exist %LIBRARY_LIB%\\AOCL-LibBlis-Win-MT-dll.lib exit 1
```
A couple things to fix here:
- I reformatted for readability
- The `.dll` file on Windows needs to be copied to `%LIBRARY_BIN%`, not `%LIBRARY_LIB%.  It appears that `cmake --install` is incorrectly copying the `.dll` to `%LIBRARY_LIB%`, so you may need to manually move it in the build script after the install step.
- We also need to test for a `.lib` file in `%LIBRARY_LIB%`.  If this file is not being generated, please add `-DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON` to the CMake arguments for the Windows build script.

---

Thanks for this. I've made the requested changes.
If I manually move the dll file to %LIBRARY_BIN%, won't that invalidate any .cmake package files that get generated? How to fix those? Ideally this would get fixed upstream...